[INFO] [2021-06-15 22:03:25,152] [3760:140577387210560] - task_executor.py[line:313]: report task 2021061522031652853221_dataio_0 0 guest 9999 to driver
[INFO] [2021-06-15 22:03:25,153] [3760:140577387210560] - control_client.py[line:42]: request update job 2021061522031652853221 task 2021061522031652853221_dataio_0 0 on guest 9999
[INFO] [2021-06-15 22:03:25,563] [3760:140577387210560] - task_executor.py[line:150]: Run 2021061522031652853221 dataio_0 2021061522031652853221_dataio_0 guest 9999 task
[INFO] [2021-06-15 22:03:25,564] [3760:140577387210560] - task_executor.py[line:151]: Component parameters on party {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': True, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense', 'need_run': True}, 'dsl_path': './examples/LDPDev/dsl/v2/hetero_secureboost/test_secureboost_train_dsl.json', 'component_parameters': {'role': {'host': {'0': {'reader_1': {'table': {'name': 'breast_hetero_host', 'namespace': 'experiment'}}, 'reader_0': {'table': {'name': 'default_credit_hetero_host', 'namespace': 'experiment'}}, 'dataio_0': {'with_label': False}, 'dataio_1': {'with_label': False}}}, 'guest': {'0': {'reader_1': {'table': {'name': 'breast_hetero_guest', 'namespace': 'experiment'}}, 'reader_0': {'table': {'name': 'default_credit_hetero_guest', 'namespace': 'experiment'}}, 'dataio_0': {'with_label': True, 'output_format': 'dense'}, 'dataio_1': {'with_label': True, 'output_format': 'dense'}}}}, 'common': {'hetero_secure_boost_0': {'task_type': 'classification', 'objective_param': {'objective': 'cross_entropy'}, 'num_trees': 3, 'validation_freqs': 1, 'encrypt_param': {'method': 'iterativeAffine'}, 'tree_param': {'max_depth': 3}}, 'evaluation_0': {'eval_type': 'binary'}}}, 'initiator': {'role': 'guest', 'party_id': 9999}, 'dsl_version': 2, 'job_parameters': {'job_type': 'predict', 'work_mode': 0, 'backend': 0, 'computing_engine': 'STANDALONE', 'federation_engine': 'STANDALONE', 'storage_engine': 'STANDALONE', 'engines_address': {'computing': {'nodes': 1, 'cores_per_node': 20}, 'federation': {'nodes': 1, 'cores_per_node': 20}, 'storage': {'nodes': 1, 'cores_per_node': 20}}, 'federated_mode': 'SINGLE', 'task_parallelism': 1, 'computing_partitions': 4, 'federated_status_collect_type': 'PULL', 'model_id': 'guest-9999#host-9998#model', 'model_version': '2021061522023585229020', 'eggroll_run': {'eggroll.session.processors.per.node': 4}, 'spark_run': {}, 'rabbitmq_run': {}, 'pulsar_run': {}, 'adaptation_parameters': {'task_memory_per_node': 0, 'if_initiator_baseline': False, 'task_cores_per_node': 4, 'request_task_cores': 4, 'task_nodes': 1}}, 'conf_path': './examples/LDPDev/test_predict_conf.json', 'role': {'host': [9998], 'guest': [9999]}, 'local': {'role': 'guest', 'party_id': 9999}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO', 'output_data_name': ['data']}
[INFO] [2021-06-15 22:03:25,564] [3760:140577387210560] - task_executor.py[line:152]: Task input dsl {'data': {'data': ['reader_0.data']}, 'model': ['pipeline.dataio_0.model']}
[INFO] [2021-06-15 22:03:25,565] [3760:140577387210560] - tracker_client.py[line:150]: Request read job 2021061522031652853221 task None None on guest 9999 data data info
[INFO] [2021-06-15 22:03:25,611] [3760:140577387210560] - task_executor.py[line:269]: load computing table use 4
[INFO] [2021-06-15 22:03:26,151] [3760:140577387210560] - pipelined_model.py[line:185]: parse DataIOMeta proto object normal
[INFO] [2021-06-15 22:03:26,158] [3760:140577387210560] - pipelined_model.py[line:185]: parse DataIOParam proto object normal
[WARNING] [2021-06-15 22:03:26,164] [3760:140577387210560] - data_io.py[line:855]: DataIO is deprecated, and will be removed in 1.7, use DataTransform module instead
[INFO] [2021-06-15 22:03:26,175] [3760:140577387210560] - data_io.py[line:118]: start to read dense data and change data to instance
[ERROR] [2021-06-15 22:03:30,679] [3760:140577387210560] - task_executor.py[line:191]: num row of input(30000) not equals to num row of output(0)
Traceback (most recent call last):
  File "/fate/python/fate_flow/operation/task_executor.py", line 168, in run_task
    run_object.run(component_parameters_on_party, task_run_args)
  File "/fate/python/federatedml/model_base.py", line 98, in run
    this_data_output = func(*real_param)
  File "/fate/python/federatedml/util/data_io.py", line 895, in transform
    return self.reader.read_data(data_inst, "transform")
  File "/fate/python/federatedml/util/data_io.py", line 148, in read_data
    data_instance = self.transform(input_data_features, input_data_labels)
  File "/fate/python/federatedml/util/io_check.py", line 38, in _func
    f"num row of input({input_count}) not equals to num row of output({output_count})")
OSError: num row of input(30000) not equals to num row of output(0)
